### PR TITLE
safeclib: init at 08112019

### PR DESCRIPTION
**Pull request by Beowulf0**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Review by doronbehar — APPROVED** [review]

**Comment by Beowulf0** [comment]
still important to me

File: pkgs/development/libraries/safeclib/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "safeclib";
+  version = "08112019";
+
+  src = fetchFromGitHub {
+    owner = "rurban";
+    repo = "${pname}";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/safeclib/default.nix:9]:
```suggestion
    repo = pname;
```

File: pkgs/development/libraries/safeclib/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  nativeBuildInputs = [
+    autoreconfHook
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/rurban/safeclib";
+    description = "An implementation of the secure C11 Annex K functions";
+    license = licenses.mit;
+    maintainers = with stdenv.lib.maintainers; [ ];
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/safeclib/default.nix:26]: Please add yourself as a maintainer.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 76451` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>safeclib</li>
  </ul>
</details>

**Comment by doronbehar** [comment]
@Beowulf0 please address the review.

**Pull request closed without merging**